pkg/bundle/bundle.go

```diff
@@ -82,14 +82,14 @@ type ImagePlatform struct {
 
 // Image describes a container image in the bundle
 type Image struct {
-	BaseImage
+	BaseImage   `mapstructure:",squash"`
```

Comment:
Nice catch here!
I do have two questions regarding this change:

- any particular reason you didn't include the `json` tag as well?
- we haven't used `squash` for fields so far -- could you, please, include a couple of sentences why we need to add it?

Thanks!

Reply:
> any particular reason you didn't include the json tag as well?
using squash causes the other structure to basically be inlined, as a result the json tag from the BaseImage struct will be used.

> we haven't used squash for fields so far -- could you, please, include a couple of sentences why we need to add it, and why it's inconsistent with all other fields in the bundle?
As described in the bug #658, not having the squash tag prevents the Image from marshalling the fields of the BaseImage struct.